Ini.pas

```diff
@@ -105,8 +105,8 @@   TContestDefinition = record
      ExchType1: etOpName;
      ExchType2: etNaQpExch2;
      ExchFieldEditable: True;
-     ExchDefault: 'MIKE OR';
-     Msg: '''<name> [<state|prov|dxcc-entity>]'' (e.g. MIKE OR)';
+     ExchDefault: 'Alex GH';
+     Msg: '''<name> [<state|prov|dxcc-entity>]'' (e.g. ALEX GH)';
```

Comment:
For NAQP, the default exchange must "ALEX ON" (capital letters, I guess + ONtario). The message (line 109) must be similar.

Reply:
Good find. Done. thank you.